lua/astrocommunity/bars-and-lines/statuscol-nvim/init.lua

```diff
@@ -1,5 +1,10 @@
 return {
-  { "luukvbaal/statuscol.nvim", lazy = false, opts = {} },
+  {
+    "luukvbaal/statuscol.nvim",
+    event = "User AstroFile",
+    lazy = vim.fn.argv()[1] == nil,
```

Comment:
I don't understand why this is here. You're checking of neovim was opened with a file, but User AstroFile does that itself.

Reply:
If you want to load the plugin if the filename is empty, I'd just swap the event out instead of having this lazy = vim.fn.... thing